.github/workflows/opampsupervisor-testbuild.yaml

```diff
@@ -0,0 +1,75 @@
+name: CI - OpAMP supervisor
+
+on:
+  merge_group: 
+  push:
+    branches: [main]
+    paths:
+      - "distributions/otelcol-contrib/**"
```

Comment:
does this really need to run when something in otelcol-contrib changes?

Reply:
removed